test/integration/checkout_int_test.go

```diff
@@ -284,9 +280,10 @@ func (suite *CheckoutIntegrationTestSuite) TestCheckoutBuildtimeClosure() {
 		e2e.OptAppendEnv(constants.DisableRuntime+"=false"),
 	)
 	// Expect the number of build deps to be 27 which is more than the number of runtime deps.
-	// Also expect libxcrypt which should not be in the runtime closure.
+	// Also expect ncurses which should not be in the runtime closure.
+	cp.Expect("Checked out project", e2e.RuntimeSourcingTimeoutOpt)
 	cp.Expect("27")
-	cp.Expect("libxcrypt")
+	cp.Expect("ncurses")
```

Comment:
I'm not sure why this changed, but in my testing this was also failing in the non-refactored version.